Kingfisher/UIImageView+Kingfisher.swift

```diff
@@ -104,12 +106,30 @@ public extension UIImageView {
     
     :returns: A task represents the retriving process.
     */
+    
     public func kf_setImageWithURL(URL: NSURL,
                       placeholderImage: UIImage?,
                            optionsInfo: KingfisherOptionsInfo?,
                          progressBlock: DownloadProgressBlock?,
                      completionHandler: CompletionHandler?) -> RetrieveImageTask
     {
+        
+        var indicator = UIActivityIndicatorView(activityIndicatorStyle:.Gray)
+        if ( showIndicatorWhenLoading == true)
+        {
```

Comment:
Curly brackets should be in the same line as the if, to follow the same coding style.